### DISCUSSION TITLE
not able to find termux files

### DISCUSSION THREAD

**Discussion by NaviTheCoderboi** [termux/termux-app#3279] [Q&A] [answered]
When i ran the `PWD` command i got this 
![IMG_20230302_212008](https://user-images.githubusercontent.com/102732282/222481292-189f7f1d-b337-4aae-b119-cffff5959601.jpg)
But when i search using my file manager i not find the data folder or termux files 
I am using file manager + and i used my mobile's own file manager too but not found 

Can you anyone tell me?

**Comment by agnostic-apollo** [comment, marked as answer]
Private app data directories of other apps are not accessible to file managers without root (in global mount namespace for android `11+`). Use termux  SAF instead.

https://wiki.termux.com/wiki/Internal_and_external_storage

**Reply by Noman964** [reply]
https://wiki.termux.com/wiki/Internal_and_external_storage
